common/test/acceptance/pages/studio/container.py

```diff
@@ -284,6 +284,12 @@ def edit(self):
         """
         return _click_edit(self, '.edit-button', '.xblock-studio_view')
 
+    def edit_visibility(self):
```

Comment:
Yup, this edit_visibility is part of ContainerPage, while the lower one is in XBlockWrapper.